pkg/prepconf/prepare.go

```diff
@@ -104,19 +104,13 @@ func generateConfig(conf Config) (*visorconfig.V1, error) {
 	//if out.STCP, err = visor.DefaultSTCPConfig(); err != nil {
 	//	return nil, err
 	//}
-	out.Dmsgpty.AuthFile = "/var/skywire-visor/dsmgpty/whitelist.json"
 	out.Dmsgpty.CLIAddr = "/run/skywire-visor/dmsgpty/cli.sock"
-	out.Transport.LogStore.Type = "file"
-	out.Transport.LogStore.Location = "/var/skywire-visor/transports"
 	out.Hypervisors = bp.HypervisorPKs
 	out.LogLevel = skyenv.DefaultLogLevel
 	out.ShutdownTimeout = visorconfig.DefaultTimeout
 	out.RestartCheckDelay = visorconfig.Duration(restart.DefaultCheckDelay)
 	out.Launcher = &visorconfig.V1Launcher{
-		Discovery: &visorconfig.V1AppDisc{
-			ServiceDisc:    skyenv.DefaultServiceDiscAddr,
-			UpdateInterval: visorconfig.Duration(skyenv.AppDiscUpdateInterval),
-		},
+		ServiceDisc: skyenv.DefaultServiceDiscAddr,
```

Comment:
There seem to be a few things missing that we currently support on `develop`. Just generate a config on develop and you will be able to compare the differences easily.

Reply:
Done.